packages/sns/candid/sns_swap.d.ts

```diff
@@ -12,12 +12,12 @@ export interface CanisterStatusResultV2 {
   controller: Principal;
   status: CanisterStatusType;
   freezing_threshold: bigint;
-  balance: Array<[Uint8Array, bigint]>;
+  balance: Array<[Uint8Array | number[], bigint]>;
```

Comment:
what version of `didc` did you use to generate these files?

I would have excepted this to not change in your PR and if they changed the way mappings for number array are generated, then generate these for all bindings in a separate PR.

Reply:
I agree

Reply:
Hmm... `didc 0.1.6`

Reply:
can you try with `v0.1.4`?

Reply:
Sure, the change was reverted. I added only a single function (using v0.1.4) to the did file because there are new fields and this break the aggregator. Not sure the aggregator should be updated by this PR.

<img width="1193" alt="image" src="https://user-images.githubusercontent.com/98811342/220547862-99739611-2e2c-4109-b4f9-77888a13f194.png">

Please let me know if you have another opinion/plan.

Reply:
this function is `convertSwap` is in NNS-dapp no?

so we can merge this PR, release ic-js and then create a PR in NNS-dapp that bump ic-js and fixes this error or I am missing something?

Reply:
I guess the error is not related to this PR but another PR that has been merged and not yet integrated in NNS-dapp?

Reply:
Yes, sorry, you are right, it's about the PR that uses changes of this PR.

Reply:
Let's do that then 👍